ngraph_bridge/ngraph_utils.h

```diff
@@ -140,32 +140,38 @@ Status ValuesFromConstNode(const NodeDef& node,
       n_elements *= shape.dim(i).size();
     }
     values->resize(n_elements);
+
+    auto val_lastsaved = 0;
+#define SET_VALS(val_size, val_vec)                        \
```

Comment:
can we use a templated function instead?

Reply:
Restructured the code